txpool/pool.go

```diff
@@ -2187,19 +2193,29 @@ func (mt *metaTx) better(than *metaTx, pendingBaseFee uint64) bool {
 
 	switch mt.currentSubPool {
 	case PendingSubPool:
-		var effectiveTip, thanEffectiveTip uint64
-		if pendingBaseFee <= mt.minFeeCap {
-			effectiveTip = cmp.Min(mt.minFeeCap-pendingBaseFee, mt.minTip)
+		var effectiveTip, thanEffectiveTip uint256.Int
+		difference := uint256.NewInt(0)
+		difference.Sub(&than.minFeeCap, &pendingBaseFee)
```

Comment:
There were 2 differences here previously:
`mt.minFeeCap-pendingBaseFee`
and
`than.minFeeCap-pendingBaseFee`